model/error.yaml

```diff
@@ -0,0 +1,34 @@
+groups:
+  - id: error
```

Comment:
There should be a markdown file for it, and probably a separate PR as this is a very important concept that should not be "hidden" in a HTTP PR.

Reply:
Agreed. I created https://github.com/open-telemetry/semantic-conventions/issues/279 to track it - I'd like to unblock HTTP semconv and also there are some efforts to create an attribute registry/dictionary  (like this one https://github.com/open-telemetry/semantic-conventions/pull/208) which would make it happen without any special effort.